Build.proj

```diff
@@ -1,13 +1,15 @@
 <Project Sdk="Microsoft.Build.Traversal">
 
   <ItemGroup>
-    <!-- Microsoft.WindowsDesktop.App shared framework -->
+    <!-- Microsoft.WindowsDesktop.App shared framework and sentinel project -->
     <ProjectReference Include="src\windowsdesktop\src\sfx\**\*.sfxproj" />
     <ProjectReference Include="src\windowsdesktop\src\bundle\**\*.bundleproj" />
     <ProjectReference Include="src\windowsdesktop\tests\**\*.Tests.csproj" />
+    <ProjectReference Include="src\windowsdesktop\src\Microsoft.WindowsDesktop.App.Internal\Microsoft.WindowsDesktop.App.Internal.csproj" />
```

Comment:
I thought about putting the project under sfx, but there are a number of Directory.Build.* files there which have properties that don't apply.

Reply:
I think putting it under `src` directly is the correct location.